src/shared/iodide-logo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { css } from 'emotion'
+import Typography from '@material-ui/core/Typography'
+
+export default class IodideLogo extends React.Component {
+  render() {
+    return (
+      <Typography
+        variant="title"
+        className={css`
+          font-family: 'Zilla Slab', serif;
+          font-size:30px;
+          font-weight: 300;`}
+      >
+        <a href={this.props.backLink} target="_self" style={{ color: 'white', textDecoration: 'none' }}>
```

Comment:
Minor style point - can we use `css` here as well, as we do in the `Typography` element?

Reply:
Sure thing

Reply:
Actually it turns out that if I change this, the logo on hover gets underlined due to this css:

https://github.com/iodide-project/iodide/blob/a9276761e9cb2f1f8364eb1da0abf8105b9e74cd/src/style/menu-and-button-and-ui-styles.css#L124

I wonder if we could just get rid of it? I think the only place it's used is in the header bar, but I don't see much difference in how things work there if I take them out.

Reply:
hmm, that's annoying, but true that it is probably only used in the header, so I'd say let's remove it.

Reply:
... one of the many, many reasons why the css-in-js paradigm is better for larger apps.

Reply:
Actually I'll make the css selector more specific so it will only be applied for links within the header.